pkg/connector/handlematrix.go

```diff
@@ -391,3 +398,112 @@ func (wa *WhatsAppClient) HandleMatrixMembership(ctx context.Context, msg *bridg
 
 	return true, nil
 }
+
+func (wa *WhatsAppClient) HandleMatrixRoomName(ctx context.Context, msg *bridgev2.MatrixRoomName) (bool, error) {
+	portalJID, err := waid.ParsePortalID(msg.Portal.ID)
+	if err != nil {
+		return false, err
+	}
+
+	if msg.Portal.RoomType == database.RoomTypeDM {
+		return false, fmt.Errorf("cannot set room name for DM")
+	}
+
+	err = wa.Client.SetGroupName(portalJID, msg.Content.Name)
+	if err != nil {
+		return false, err
+	}
+
+	return true, nil
+}
+
+func (wa *WhatsAppClient) HandleMatrixRoomTopic(ctx context.Context, msg *bridgev2.MatrixRoomTopic) (bool, error) {
+	portalJID, err := waid.ParsePortalID(msg.Portal.ID)
+	if err != nil {
+		return false, err
+	}
+
+	if msg.Portal.RoomType == database.RoomTypeDM {
+		return false, fmt.Errorf("cannot set room topic for DM")
+	}
+
+	err = wa.Client.SetGroupTopic(portalJID, "", "", msg.Content.Topic)
```

Comment:
It would be more proper to store the topic ID in portal metadata so the appropriate values can be passed here